src/homeform.cpp

```diff
@@ -1963,9 +1972,14 @@ void homeform::update() {
         pidHR->setValue(QString::number(treadmill_pid_heart_zone));
 
         if (trainProgram) {
-            peloton_offset->setValue(QString::number(trainProgram->offsetElapsedTime()) + QStringLiteral(" sec."));
+            peloton_offset->setValue(trainProgram->remainingTime().toString("h:mm:ss"));
+            peloton_offset->setSecondLine(QString::number(trainProgram->offsetElapsedTime()) + QStringLiteral(" sec."));
```

Comment:
Yes I guess the best way to conserve the legacy is to create a new one. We need to play safely with changes because a lot of people are involved on this ;) thanks